File: 백준/Bronze/10768. 특별한 날/특별한 날.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args) {
       Scanner scan = new Scanner(System.in);
       
       int A = scan.nextInt();
       int B = scan.nextInt();
       
       if(A==2 && B==18) System.out.println("Special");
       else if(A < 2) {
    	   System.out.println("Before");
       }else if(A > 2){
    	   System.out.println("After");
       }else {
    	   if(B < 18) {
    		   System.out.println("Before");
    	   }else if(B > 18) {
    		   System.out.println("After");
    	   }
       }
    } 
}

File: 백준/Bronze/10768. 특별한 날/특별한 날.py
A = int(input())
B = int(input())


if(A==2 and B==18): print("Special")
elif(A < 2): print("Before")
elif(A > 2): print("After")
else:
    if(B < 18): print("Before")
    elif(B > 18): print("After")